source/common/mongo/bson_impl.cc

```diff
@@ -10,6 +10,15 @@
 
 #include "spdlog/spdlog.h"
 
+#ifdef __APPLE__
```

Comment:
This will end up getting used again elsewhere. Can we put this into common/common/byte_order.h or something?